### PR TITLE
More api

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: nb/api/Polyhedron.nb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+md`
+### Polyhedron()
+Parameter|Default|Type
+---|---|---
+paths|[]|A list of loop of points.
+
+Constructs a potentially open surface.
+
+We use loops of points rather than indexed paths since converting indexed paths to loops of points is straight-forward, but the inverse requires deduplication.
```
Comment (BarbourSmith) [inline nb/api/Polyhedron.nb:9]: 👍 This is nice and clear I think

File: nb/api/remesh.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+### remesh()
+Parameter|Default|Type
+---|---|---
```
Comment (BarbourSmith) [inline nb/api/remesh.md:3]: I like this new structured approach to the information

File: nb/api/seam.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+### seam()
+Parameter|Default|Type
+---|---|---
+...selections||Shapes to select the areas to seam.
+
+Add non-geometric edges where the selections intersect the shape.
```
Comment (BarbourSmith) [inline nb/api/seam.md:6]: I understand what this does from the notes/examples, but I don't really understand why one would want to do it 😛

File: nb/api/section.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    'Orb(10).section(XY().seq({ from: -5, to: 5, by: 3 }, z)) cuts horizontal sections through a sphere.'
+  );
+```
+
+![Image](section.md.0.png)
+
+Orb(10).section(XY().seq({ from: -5, to: 5, by: 3 }, z)) cuts horizontal sections through a sphere.
+
+```JavaScript
+Box(10, 10, 10)
```
Comment (BarbourSmith) [inline nb/api/section.md:20]: This is a beautiful example.

**Review by BarbourSmith — APPROVED** [review]

File: nb/api/seam.md (hunk 3 above)
Comment (pentacular) [reply nb/api/seam.md:6]: Yeah, it has relatively limited application.

**Pull request merged**